test/token-pool-proxy.test.js

```diff
@@ -53,4 +94,156 @@ contract('Token Pool', accounts => {
       expect(tokenAddr).to.be.equal(ctx.contracts.token.address)
     })
   })
+
+  describe('sets token pool reward', () => {
+    it("doesn't set 0", async () => {
+      const tx = ctx.contracts.pool.setTokenPoolReward(0, {from: ctx.actors.mfg})
+      await errors.expectRevert(tx)
+    })
+    it('sets new reward', async () => {
+      const tx = await ctx.contracts.pool.setTokenPoolReward(10, {from: ctx.actors.mfg})
+
+      expect(tx.logs.length).to.be.equal(1)
+      const log = tx.logs[0]
+      expect(log.event).to.be.equal('TokenPoolRewardUpdated')
+    })
+    it('does not set same reward as previously set', async () => {
+      await ctx.contracts.pool.setTokenPoolReward(10, {from: ctx.actors.mfg})
+
+      const tx = ctx.contracts.pool.setTokenPoolReward(10, {from: ctx.actors.mfg})
+      await errors.expectRevert(tx)
+    })
+  })
+
+  describe('change manufacturer wallet', () => {
+    it('new address cannot be 0x0', async () => {
+      const fn = ctx.contracts.pool.changeManufacturerWallet(0x0, {from: ctx.actors.mfg})
+      await errors.expectRevert(fn)
+    })
+    it('must be a manufacturer', async () => {
+      await ctx.contracts.member.addNetworkMember(ctx.actors.admin, false, false, false, 'test_pools', {from: ctx.actors.owner})
```

Comment:
you can delete this line, not needed for the test, since alice is calling the function and is not a mfg.  Its not relevant who the new mfg is.

Reply:
updated